src/shell/nushell.rs

```diff
@@ -0,0 +1,26 @@
+use super::shell::Shell;
+use std::path::Path;
+
+#[derive(Debug)]
+pub struct Nushell;
+
+impl Shell for Nushell {
+    fn to_clap_shell(&self) -> clap_complete::Shell {
+        panic!("Shell completion is not supported for Nushell (yet.)");
```

Comment:
Can we link to the issue in the Clap repo?
* https://github.com/clap-rs/clap/issues/2778